src/Task/GherkinLint.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+declare(strict_types=1);
+
+namespace GrumPHP\Task;
+
+use GrumPHP\Formatter\ProcessFormatterInterface;
+use GrumPHP\Runner\TaskResult;
+use GrumPHP\Runner\TaskResultInterface;
+use GrumPHP\Task\Config\ConfigOptionsResolver;
+use GrumPHP\Task\Context\ContextInterface;
+use GrumPHP\Task\Context\GitPreCommitContext;
+use GrumPHP\Task\Context\RunContext;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * @extends AbstractExternalTask<ProcessFormatterInterface>
+ */
+class GherkinLint extends AbstractExternalTask
+{
+    public static function getConfigurableOptions(): ConfigOptionsResolver
+    {
+        $resolver = new OptionsResolver();
+        $resolver->setDefaults([
+            'directory' => 'features',
```

Comment:
I noticed that if the specified directory doesn't exist, it fails with:
```
In FeatureFinder.php line 25:
  Provided path "/var/www/html/features" does not exist

lint <path>
Failed to run grumphp run --tasks=gherkin_lint: exit status 1
```

Is this expected ? or should we skip in such cases? 
How are other tasks handling these cases?

Reply:
Yes, it's expected. For example src/Task/Gherkin.php has the same configuration, and returns the same error.